taxi/forms.py

```diff
@@ -0,0 +1,31 @@
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+from django import forms
+
+from taxi.models import Driver, Car
+from taxi.validators import DriverValidationLicenseNumber
+
+
+class DriverCreationForm(UserCreationForm, DriverValidationLicenseNumber):
+    class Meta(UserCreationForm.Meta):
+        model = Driver
+        fields = (UserCreationForm.Meta.fields
+                  + ("first_name", "last_name", "license_number"))
+
+
+class DriverLicenseUpdateForm(forms.ModelForm, DriverValidationLicenseNumber):
+    class Meta:
+        model = Driver
+        fields = ("license_number",)
+
+
+class CarForm(forms.ModelForm):
+    drivers = forms.ModelMultipleChoiceField(
+        queryset=Driver.objects.all(),
```

Comment:
The queryset for the `drivers` field should be filtered to include only users who are drivers. Consider using a filter like `Driver.objects.filter(is_driver=True)` if such a field exists.